zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3068,6 +3112,138 @@ components:
         outputValue:
           description: The value of the evaluated decision output.
           type: string
+    DecisionInstanceSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/DecisionInstanceFilterRequest"
+    DecisionInstanceFilterRequest:
+      type: object
+      properties:
+        key:
+          type: integer
+          format: int64
+          description: The key of the decision instance.
+        state:
+          $ref: '#/components/schemas/DecisionInstanceStateEnum'
+          description: The state of the decision instance.
+        evaluationDateBefore:
```

Comment:
This is more of a question - was it agreed to use DateBefore and DateAfter for date filters ? Because I also have dates for example in process instance search (startDate and endDate). FYI @ralfpuchert

Reply:
@ralfpuchert I see that in your implementation for [IncidentFilterRequest](https://github.com/camunda/camunda/pull/22418/files#diff-a18635d76659a781a6e8d301bd0fed7c77fd6cc5c8cfc57e11c5bbc19c78c9cdR2156), you do not use DateBefore and DateAfter, for creationTime for example. So which version should we use after all ? Thank you